xml/1.4/modules/ROOT/pages/xml-reference.adoc

```diff
@@ -0,0 +1,263 @@
+= XML Module Reference - Mule 4
+:page-aliases: connectors::xml/xml-reference.adoc
+
+
+
+XML Module v1.4
+
+The XML Module can process and extract data from an XML document. This module should be used for advanced use cases or use cases associated with XML standards such as XSLT, XPath, and XQuery or XSD. DataWeave is not only perfectly capable to handle most of those use cases and can replace those standards the majority of the times, but it's also our recommended approach.
+
+Release Notes: xref:release-notes::mule-runtime/module-xml.adoc[XML Module Release Notes]
+
+== Configurations
+---
+[[config]]
+=== Default Configuration
+
+
+Default configuration
+
+
+==== Parameters
+[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | |x
+| Namespaces a| Array of <<namespace>> |  Allows to globally map prefixes to namespace URIs. On this version, the xpath-extract operation is the only one to consume these mappings |  |
+| Expand Entities a| Enumeration, one of:
+
+** NEVER
+** INTERNAL
+** ALL |  Defines how to treat entity expansion. Setting a value different than NEVER renders the application vulnerable to XXE and/or DoS attacks |  NEVER |
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime actually purges the instances when it sees fit. |  |
+|===
+
+
+== Operations
+
+* <<validateSchema>>
+* <<xpathExtract>>
+* <<xqueryTransform>>
+* <<xsltTransform>>
+
+
+[[validateSchema]]
+=== Validate Schema
+`<xml-module:validate-schema>`
+
+
+Validates that the input content is compliant with a given schema. This operation supports referencing many schemas (using comma as a separator) which include each other.
+
+
+==== Parameters
+[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | |x
+| Schemas a| String |  The path to the schema file. You can specify multiple schema locations by using a comma separator |  |x
```

Comment:
Just have a question. With the new feature, couldn’t you provide only the Schema Content to validate? And in that case, why is the Schemas parameter required?